SPECS/helm/helm.spec

```diff
@@ -24,16 +24,18 @@ Source0:       https://github.com/helm/helm/archive/refs/tags/v%{version}.tar.gz
 #           -cf %%{name}-%%{version}-vendor.tar.gz vendor
 #
 Source1:       %{name}-%{version}-vendor.tar.gz
+Patch0:        CVE-2024-45338.patch
 BuildRequires: golang
 
 %description
 Helm is a tool that streamlines installing and managing Kubernetes applications. Think of it like apt/yum/homebrew for Kubernetes.
 
 %prep
-%autosetup -p1
```

Comment:
here also, replaced -p1 with -N option here, so that patch will be applied to SOURCE1 as well (via autopatch below)